station/app/api/api_v1/endpoints/station_status.py

```diff
@@ -8,7 +8,7 @@
 import psutil
 
 # todo singleton minio client
-minio_client = MinioClient()
+#minio_client = MinioClient()
```

Comment:
mark all instances where this is commented out with a todo or try to fix initialization i.e. initialize client in function call and not as global object